tpl-implementations/include/experimental/__p1673_bits/kokkos-kernels/blas1_vector_abs_sum_kk.hpp

```diff
@@ -2,20 +2,39 @@
 #ifndef LINALG_TPLIMPLEMENTATIONS_INCLUDE_EXPERIMENTAL_P1673_BITS_KOKKOSKERNELS_VEC_ABS_SUM_HPP_
 #define LINALG_TPLIMPLEMENTATIONS_INCLUDE_EXPERIMENTAL_P1673_BITS_KOKKOSKERNELS_VEC_ABS_SUM_HPP_
 
+#include "signal_kokkos_impl_called.hpp"
+
 namespace KokkosKernelsSTD {
 
-template<class ExecSpace,
+// keeping this in mind: https://github.com/kokkos/stdBLAS/issues/122
+
+template<class ExeSpace,
          class ElementType,
 	 std::experimental::extents<>::size_type ext0,
          class Layout,
-         class Accessor,
          class Scalar>
-Scalar vector_abs_sum(kokkos_exec<ExecSpace>,
-		      std::experimental::mdspan<ElementType, std::experimental::extents<ext0>, Layout, Accessor> v,
+Scalar vector_abs_sum(kokkos_exec<ExeSpace> /*kexe*/,
```

Comment:
I don't think currently kokkos_exec stores an instance (see https://github.com/kokkos/stdBLAS/blob/main/tpl-implementations/include/experimental/__p1673_bits/kokkos-kernels/exec_policy_wrapper_kk.hpp) so that is why I am using `ExeSpace()` for the range policy